### ISSUE TITLE
"Symbol not found" when running grunt-sassdoc which uses babel-runtime

### ISSUE THREAD

**Issue by sdec** [babel/babel#1242]
When running grunt-sassdoc, which uses babel, we get the following error for all babel-runtime versions above 5.0.12 (for 5.0.12 it does work):

```>> ReferenceError: Symbol is not defined```

Stack trace:

```
[4mInitializing[24m
Command-line options: [36m--verbose[39m

Reading "Gruntfile.coffee" Gruntfile...[32mOK[39m

[1mRegistering Gruntfile tasks.[22m

[1mRegistering "grunt-bless" local Npm module tasks.[22m
Reading /Users/sanderdecoster/local_projects/orc-component-library/node_modules/grunt-bless/package.json...[32mOK[39m
Parsing /Users/sanderdecoster/local_projects/orc-component-library/node_modules/grunt-bless/package.json...[32mOK[39m
Loading "bless.js" tasks...[32mOK[39m
+ [36mbless[39m

[1mRegistering "grunt-bower-task" local Npm module tasks.[22m
Reading /Users/sanderdecoster/local_projects/orc-component-library/node_modules/grunt-bower-task/package.json...[32mOK[39m
Parsing /Users/sanderdecoster/local_projects/orc-component-library/node_modules/grunt-bower-task/package.json...[32mOK[39m
Loading "bower_task.js" tasks...[32mOK[39m
+ [36mbower[39m

[1mRegistering "grunt-bushcaster" local Npm module tasks.[22m
Reading /Users/sanderdecoster/local_projects/orc-component-library/node_modules/grunt-bushcaster/package.json...[32mOK[39m
Parsing /Users/sanderdecoster/local_projects/orc-component-library/node_modules/grunt-bushcaster/package.json...[32mOK[39m
Loading "bushcaster.js" tasks...[32mOK[39m
+ [36mbushcaster[39m

[1mRegistering "grunt-coffeelint" local Npm module tasks.[22m
Reading /Users/sanderdecoster/local_projects/orc-component-library/node_modules/grunt-coffeelint/package.json...[32mOK[39m
Parsing /Users/sanderdecoster/local_projects/orc-component-library/node_modules/grunt-coffeelint/package.json...[32mOK[39m
Loading "coffeelint.js" tasks...[32mOK[39m
+ [36mcoffeelint[39m

[1mRegistering "grunt-contrib-clean" local Npm module tasks.[22m
Reading /Users/sanderdecoster/local_projects/orc-component-library/node_modules/grunt-contrib-clean/package.json...[32mOK[39m
Parsing /Users/sanderdecoster/local_projects/orc-component-library/node_modules/grunt-contrib-clean/package.json...[32mOK[39m
Loading "clean.js" tasks...[32mOK[39m
+ [36mclean[39m

[1mRegistering "grunt-contrib-coffee" local Npm module tasks.[22m
Reading /Users/sanderdecoster/local_projects/orc-component-library/node_modules/grunt-contrib-coffee/package.json...[32mOK[39m
Parsing /Users/sanderdecoster/local_projects/orc-component-library/node_modules/grunt-contrib-coffee/package.json...[32mOK[39m
Loading "coffee.js" tasks...[32mOK[39m
+ [36mcoffee[39m

[1mRegistering "grunt-contrib-compass" local Npm module tasks.[22m
Reading /Users/sanderdecoster/local_projects/orc-component-library/node_modules/grunt-contrib-compass/package.json...[32mOK[39m
Parsing /Users/sanderdecoster/local_projects/orc-component-library/node_modules/grunt-contrib-compass/package.json...[32mOK[39m
Loading "compass.js" tasks...[32mOK[39m
+ [36mcompass[39m

[1mRegistering "grunt-contrib-concat" local Npm module tasks.[22m
Reading /Users/sanderdecoster/local_projects/orc-component-library/node_modules/grunt-contrib-concat/package.json...[32mOK[39m
Parsing /Users/sanderdecoster/local_projects/orc-component-library/node_modules/grunt-contrib-concat/package.json...[32mOK[39m
Loading "concat.js" tasks...[32mOK[39m
+ [36mconcat[39m

[1mRegistering "grunt-contrib-connect" local Npm module tasks.[22m
Reading /Users/sanderdecoster/local_projects/orc-component-library/node_modules/grunt-contrib-connect/package.json...[32mOK[39m
Parsing /Users/sanderdecoster/local_projects/orc-component-library/node_modules/grunt-contrib-connect/package.json...[32mOK[39m
Loading "connect.js" tasks...[32mOK[39m
+ [36mconnect[39m

[1mRegistering "grunt-contrib-copy" local Npm module tasks.[22m
Reading /Users/sanderdecoster/local_projects/orc-component-library/node_modules/grunt-contrib-copy/package.json...[32mOK[39m
Parsing /Users/sanderdecoster/local_projects/orc-component-library/node_modules/grunt-contrib-copy/package.json...[32mOK[39m
Loading "copy.js" tasks...[32mOK[39m
+ [36mcopy[39m

[1mRegistering "grunt-contrib-csslint" local Npm module tasks.[22m
Reading /Users/sanderdecoster/local_projects/orc-component-library/node_modules/grunt-contrib-csslint/package.json...[32mOK[39m
Parsing /Users/sanderdecoster/local_projects/orc-component-library/node_modules/grunt-contrib-csslint/package.json...[32mOK[39m
Loading "csslint.js" tasks...[32mOK[39m
+ [36mcsslint[39m

[1mRegistering "grunt-contrib-cssmin" local Npm module tasks.[22m
Reading /Users/sanderdecoster/local_projects/orc-component-library/node_modules/grunt-contrib-cssmin/package.json...[32mOK[39m
Parsing /Users/sanderdecoster/local_projects/orc-component-library/node_modules/grunt-contrib-cssmin/package.json...[32mOK[39m
Loading "cssmin.js" tasks...[32mOK[39m
+ [36mcssmin[39m

[1mRegistering "grunt-contrib-jade" local Npm module tasks.[22m
Reading /Users/sanderdecoster/local_projects/orc-component-library/node_modules/grunt-contrib-jade/package.json...[32mOK[39m
Parsing /Users/sanderdecoster/local_projects/orc-component-library/node_modules/grunt-contrib-jade/package.json...[32mOK[39m
Loading "jade.js" tasks...[32mOK[39m
+ [36mjade[39m

[1mRegistering "grunt-contrib-jshint" local Npm module tasks.[22m
Reading /Users/sanderdecoster/local_projects/orc-component-library/node_modules/grunt-contrib-jshint/package.json...[32mOK[39m
Parsing /Users/sanderdecoster/local_projects/orc-component-library/node_modules/grunt-contrib-jshint/package.json...[32mOK[39m
Loading "jshint.js" tasks...[32mOK[39m
+ [36mjshint[39m

[1mRegistering "grunt-contrib-requirejs" local Npm module tasks.[22m
Reading /Users/sanderdecoster/local_projects/orc-component-library/node_modules/grunt-contrib-requirejs/package.json...[32mOK[39m
Parsing /Users/sanderdecoster/local_projects/orc-component-library/node_modules/grunt-contrib-requirejs/package.json...[32mOK[39m
Loading "requirejs.js" tasks...[32mOK[39m
+ [36mrequirejs[39m

[1mRegistering "grunt-contrib-uglify" local Npm module tasks.[22m
Reading /Users/sanderdecoster/local_projects/orc-component-library/node_modules/grunt-contrib-uglify/package.json...[32mOK[39m
Parsing /Users/sanderdecoster/local_projects/orc-component-library/node_modules/grunt-contrib-uglify/package.json...[32mOK[39m
Loading "uglify.js" tasks...[32mOK[39m
+ [36muglify[39m

[1mRegistering "grunt-contrib-watch" local Npm module tasks.[22m
Reading /Users/sanderdecoster/local_projects/orc-component-library/node_modules/grunt-contrib-watch/package.json...[32mOK[39m
Parsing /Users/sanderdecoster/local_projects/orc-component-library/node_modules/grunt-contrib-watch/package.json...[32mOK[39m
Loading "watch.js" tasks...[32mOK[39m
+ [36mwatch[39m

[1mRegistering "grunt-git" local Npm module tasks.[22m
Reading /Users/sanderdecoster/local_projects/orc-component-library/node_modules/grunt-git/package.json...[32mOK[39m
Parsing /Users/sanderdecoster/local_projects/orc-component-library/node_modules/grunt-git/package.json...[32mOK[39m
Loading "git.js" tasks...[32mOK[39m
+ [36mgitadd[39m, [36mgitarchive[39m, [36mgitcheckout[39m, [36mgitclean[39m, [36mgitclone[39m, [36mgitcommit[39m, [36mgitfetch[39m, [36mgitlog[39m, [36mgitmerge[39m, [36mgitpull[39m, [36mgitpush[39m, [36mgitrebase[39m, [36mgitreset[39m, [36mgitrm[39m, [36mgitstash[39m, [36mgittag[39m

[1mRegistering "grunt-html2js" local Npm module tasks.[22m
Reading /Users/sanderdecoster/local_projects/orc-component-library/node_modules/grunt-html2js/package.json...[32mOK[39m
Parsing /Users/sanderdecoster/local_projects/orc-component-library/node_modules/grunt-html2js/package.json...[32mOK[39m
Loading "html2js.js" tasks...[32mOK[39m
+ [36mhtml2js[39m

[1mRegistering "grunt-include-replace" local Npm module tasks.[22m
Reading /Users/sanderdecoster/local_projects/orc-component-library/node_modules/grunt-include-replace/package.json...[32mOK[39m
Parsing /Users/sanderdecoster/local_projects/orc-component-library/node_modules/grunt-include-replace/package.json...[32mOK[39m
Loading "includereplace.js" tasks...[32mOK[39m
+ [36mincludereplace[39m

[1mRegistering "grunt-karma" local Npm module tasks.[22m
Reading /Users/sanderdecoster/local_projects/orc-component-library/node_modules/grunt-karma/package.json...[32mOK[39m
Parsing /Users/sanderdecoster/local_projects/orc-component-library/node_modules/grunt-karma/package.json...[32mOK[39m
Loading "grunt-karma.js" tasks...[32mOK[39m
+ [36mkarma[39m

[1mRegistering "grunt-protractor-runner" local Npm module tasks.[22m
Reading /Users/sanderdecoster/local_projects/orc-component-library/node_modules/grunt-protractor-runner/package.json...[32mOK[39m
Parsing /Users/sanderdecoster/local_projects/orc-component-library/node_modules/grunt-protractor-runner/package.json...[32mOK[39m
Loading "protractor_runner.js" tasks...[32mOK[39m
+ [36mprotractor[39m

[1mRegistering "grunt-replace" local Npm module tasks.[22m
Reading /Users/sanderdecoster/local_projects/orc-component-library/node_modules/grunt-replace/package.json...[32mOK[39m
Parsing /Users/sanderdecoster/local_projects/orc-component-library/node_modules/grunt-replace/package.json...[32mOK[39m
Loading "replace.js" tasks...[32mOK[39m
+ [36mreplace[39m

[1mRegistering "grunt-sassdoc" local Npm module tasks.[22m
Reading /Users/sanderdecoster/local_projects/orc-component-library/node_modules/grunt-sassdoc/package.json...[32mOK[39m
Parsing /Users/sanderdecoster/local_projects/orc-component-library/node_modules/grunt-sassdoc/package.json...[32mOK[39m
Loading "sassdoc.js" tasks...[31mERROR[39m
[31m>> [39mReferenceError: Symbol is not defined[31m
>> [39m  at /Users/sanderdecoster/local_projects/orc-component-library/node_modules/grunt-sassdoc/node_modules/sassdoc/node_modules/babel-runtime/regenerator/runtime.js:22:57[31m
>> [39m  at Object.<anonymous> (/Users/sanderdecoster/local_projects/orc-component-library/node_modules/grunt-sassdoc/node_modules/sassdoc/node_modules/babel-runtime/regenerator/runtime.js:511:3)[31m
>> [39m  at Module._compile (module.js:456:26)[31m
>> [39m  at Object.Module._extensions..js (module.js:474:10)[31m
>> [39m  at Module.load (/Users/sanderdecoster/local_projects/orc-component-library/node_modules/grunt-replace/node_modules/applause/node_modules/cson/node_modules/coffee-script/lib/coffee-script/register.js:45:36)[31m
>> [39m  at Function.Module._load (module.js:312:12)[31m
>> [39m  at Module.require (module.js:364:17)[31m
>> [39m  at require (module.js:380:17)[31m
>> [39m  at Object.<anonymous> (/Users/sanderdecoster/local_projects/orc-component-library/node_modules/grunt-sassdoc/node_modules/sassdoc/node_modules/babel-runtime/regenerator/index.js:12:18)[31m
>> [39m  at Module._compile (module.js:456:26)[31m
>> [39m  at Object.Module._extensions..js (module.js:474:10)[31m
>> [39m  at Module.load (/Users/sanderdecoster/local_projects/orc-component-library/node_modules/grunt-replace/node_modules/applause/node_modules/cson/node_modules/coffee-script/lib/coffee-script/register.js:45:36)[31m
>> [39m  at Function.Module._load (module.js:312:12)[31m
>> [39m  at Module.require (module.js:364:17)[31m
>> [39m  at require (module.js:380:17)[31m
>> [39m  at Object.<anonymous> (/Users/sanderdecoster/local_projects/orc-component-library/node_modules/grunt-sassdoc/node_modules/sassdoc/dist/sassdoc.js:5:27)[31m
>> [39m  at Module._compile (module.js:456:26)[31m
>> [39m  at Object.Module._extensions..js (module.js:474:10)[31m
>> [39m  at Module.load (/Users/sanderdecoster/local_projects/orc-component-library/node_modules/grunt-replace/node_modules/applause/node_modules/cson/node_modules/coffee-script/lib/coffee-script/register.js:45:36)[31m
>> [39m  at Function.Module._load (module.js:312:12)[31m
>> [39m  at Module.require (module.js:364:17)[31m
>> [39m  at require (module.js:380:17)[31m
>> [39m  at Object.<anonymous> (/Users/sanderdecoster/local_projects/orc-component-library/node_modules/grunt-sassdoc/node_modules/sassdoc/index.js:3:15)[31m
>> [39m  at Module._compile (module.js:456:26)[31m
>> [39m  at Object.Module._extensions..js (module.js:474:10)[31m
>> [39m  at Module.load (/Users/sanderdecoster/local_projects/orc-component-library/node_modules/grunt-replace/node_modules/applause/node_modules/cson/node_modules/coffee-script/lib/coffee-script/register.js:45:36)[31m
>> [39m  at Function.Module._load (module.js:312:12)[31m
>> [39m  at Module.require (module.js:364:17)[31m
>> [39m  at require (module.js:380:17)[31m
>> [39m  at Object.<anonymous> (/Users/sanderdecoster/local_projects/orc-component-library/node_modules/grunt-sassdoc/tasks/sassdoc.js:10:15)[31m
>> [39m  at Module._compile (module.js:456:26)[31m
>> [39m  at Object.Module._extensions..js (module.js:474:10)[31m
>> [39m  at Module.load (/Users/sanderdecoster/local_projects/orc-component-library/node_modules/grunt-replace/node_modules/applause/node_modules/cson/node_modules/coffee-script/lib/coffee-script/register.js:45:36)[31m
>> [39m  at Function.Module._load (module.js:312:12)[31m
>> [39m  at Module.require (module.js:364:17)[31m
>> [39m  at require (module.js:380:17)[31m
>> [39m  at loadTask (/Users/sanderdecoster/local_projects/orc-component-library/node_modules/grunt/lib/grunt/task.js:323:10)[31m
>> [39m  at /Users/sanderdecoster/local_projects/orc-component-library/node_modules/grunt/lib/grunt/task.js:361:7[31m
>> [39m  at Array.forEach (native)[31m
>> [39m  at loadTasks (/Users/sanderdecoster/local_projects/orc-component-library/node_modules/grunt/lib/grunt/task.js:360:11)[31m
>> [39m  at Task.task.loadNpmTasks (/Users/sanderdecoster/local_projects/orc-component-library/node_modules/grunt/lib/grunt/task.js:408:5)[31m
>> [39m  at Array.forEach (native)[31m
>> [39m  at module.exports (/Users/sanderdecoster/local_projects/orc-component-library/node_modules/load-grunt-tasks/load-grunt-tasks.js:27:29)[31m
>> [39m  at Object.module.exports (/Users/sanderdecoster/local_projects/orc-component-library/Gruntfile.coffee:8:9)[31m
>> [39m  at loadTask (/Users/sanderdecoster/local_projects/orc-component-library/node_modules/grunt/lib/grunt/task.js:325:10)[31m
>> [39m  at Task.task.init (/Users/sanderdecoster/local_projects/orc-component-library/node_modules/grunt/lib/grunt/task.js:437:5)[31m
>> [39m  at Object.grunt.tasks (/Users/sanderdecoster/local_projects/orc-component-library/node_modules/grunt/lib/grunt.js:120:8)[31m
>> [39m  at Object.module.exports [as cli] (/Users/sanderdecoster/local_projects/orc-component-library/node_modules/grunt/lib/grunt/cli.js:38:9)[31m
>> [39m  at Object.<anonymous> (/usr/local/lib/node_modules/grunt-cli/bin/grunt:45:20)[31m
>> [39m  at Module._compile (module.js:456:26)[31m
>> [39m  at Object.Module._extensions..js (module.js:474:10)[31m
>> [39m  at Module.load (module.js:356:32)[31m
>> [39m  at Function.Module._load (module.js:312:12)[31m
>> [39m  at Function.Module.runMain (module.js:497:10)[31m
>> [39m  at startup (node.js:119:16)[31m
>> [39m  at node.js:906:3

[1mRegistering "grunt-scss-lint" local Npm module tasks.[22m
Reading /Users/sanderdecoster/local_projects/orc-component-library/node_modules/grunt-scss-lint/package.json...[32mOK[39m
Parsing /Users/sanderdecoster/local_projects/orc-component-library/node_modules/grunt-scss-lint/package.json...[32mOK[39m
Loading "scss-lint.js" tasks...[32mOK[39m
+ [36mscsslint[39m

[1mRegistering "grunt-usemin" local Npm module tasks.[22m
Reading /Users/sanderdecoster/local_projects/orc-component-library/node_modules/grunt-usemin/package.json...[32mOK[39m
Parsing /Users/sanderdecoster/local_projects/orc-component-library/node_modules/grunt-usemin/package.json...[32mOK[39m
Loading "usemin.js" tasks...[32mOK[39m
+ [36musemin[39m, [36museminPrepare[39m

[1mRegistering "grunt-wiredep" local Npm module tasks.[22m
Reading /Users/sanderdecoster/local_projects/orc-component-library/node_modules/grunt-wiredep/package.json...[32mOK[39m
Parsing /Users/sanderdecoster/local_projects/orc-component-library/node_modules/grunt-wiredep/package.json...[32mOK[39m
Loading "wiredep.js" tasks...[32mOK[39m
+ [36mwiredep[39m

[1mRegistering "grunt-wrap" local Npm module tasks.[22m
Reading /Users/sanderdecoster/local_projects/orc-component-library/node_modules/grunt-wrap/package.json...[32mOK[39m
Parsing /Users/sanderdecoster/local_projects/orc-component-library/node_modules/grunt-wrap/package.json...[32mOK[39m
Loading "wrap.js" tasks...[32mOK[39m
+ [36mwrap[39m

[1mLoading grunt configs via "load-grunt-configs" from 15 file(s).[22m
Loading grunt/clean.coffee...[32mOK[39m
+ clean: [tmp, dist, sassCache, node, bower, tmpScripts, tmpComponentCss, tmpComponentMixins, orchid]

Loading grunt/coffee.coffee...[32mOK[39m
+ coffee: [compile]

Loading grunt/compass.coffee...[32mOK[39m
+ compass: [compile]

Loading grunt/concat.coffee...is fn(), invoking...[32mOK[39m
+ concat: [orchid, orchid_base, orchid_large, orchid_medium, orchid_small, orchid_theme_palo, orchid_theme_palo_base, orchid_theme_palo_large, orchid_theme_palo_medium, orchid_theme_palo_small, orchid_theme_pb, orchid_theme_pb_base, orchid_theme_pb_large, orchid_theme_pb_medium, orchid_theme_pb_small, orchid_normalize, mixins]

Loading grunt/connect.coffee...[32mOK[39m
+ connect: [options, release, development]

Loading grunt/copy.coffee...[32mOK[39m
+ copy: [svg, img, componentCss, componentMixinsJade, componentMixinsCss, componentTemplatesJade, componentFonts, orchid, dist, minifiedCss]

Loading grunt/cssmin.coffee...is fn(), invoking...[32mOK[39m
+ cssmin: [tmpLibCss]

Loading grunt/includereplace.coffee...is fn(), invoking...[32mOK[39m
+ includereplace: [tmp]

Loading grunt/jade.coffee...[32mOK[39m
+ jade: [compile]

Loading grunt/remove-include-tags.coffee...is fn(), invoking...[32mOK[39m
+ remove-include-tags: [dev]

Loading grunt/requirejs.coffee...is fn(), invoking...[32mOK[39m
+ requirejs: [compile]

Loading grunt/sassdoc.coffee...is fn(), invoking...[32mOK[39m
+ sassdoc: [default]

Loading grunt/scsslint.coffee...is fn(), invoking...[32mOK[39m
+ scsslint: [options, default]

Loading grunt/uglify.coffee...is fn(), invoking...[32mOK[39m
+ uglify: [tmpLibJs]

Loading grunt/watch.coffee...[32mOK[39m
+ watch: [sass, sassdoc, coffee, jade, reload]

Initializing config...[32mOK[39m
Loading "Gruntfile.coffee" tasks...[32mOK[39m
+ [36mbuild[39m, [36mchangelog[39m, [36mcleanAll[39m, [36mdefault[39m, [36mrelease[39m, [36mremove-include-tags[39m, [36mserve[39m, [36mserve:release[39m, [36mupdate-orchid[39m, [36mversions[39m

Running tasks: [36msassdoc[39m
[33mWarning: Task "sassdoc" not found. Use --force to continue.[39m

[31mAborted due to warnings.[39m

```

**Comment by sebmck** [comment]
Source file please.

**Comment by sebmck** [comment]
Oh wait, nevermind.

**Comment by sdec** [comment]
This is our sassdoc configuration:

```
module.exports = ->
  default:
    src: '<%= paths.node %>/orc-core/**/*.scss'
    options:
      dest: '<%= paths.tmp %>/sassdoc'
      display:
        access: ['public', 'private']
        alias: true
```

**Comment by sebmck** [comment]
Fixed as of 5.1.1, thanks!

**Comment by sdec** [comment]
No problem! Thank you for the quick fix.
